cpp/src/example/example.cc

```diff
@@ -0,0 +1,87 @@
+#include "example.h"
+#include <execinfo.h>
+#include <ray/api.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <chrono>
+#include <iostream>
+#include <thread>
+
+using namespace ray::api;
+
+void handler(int sig) {
+  void *array[10];
+  size_t size;
+  size = backtrace(array, 10);
+  backtrace_symbols_fd(array, size, STDERR_FILENO);
+  exit(1);
+}
+
+int Foo0() { return 1; }
+int Foo(int x) { return x + 1; }
+
+int Bar(int x, int y) { return x + y; }
+
+int main() {
+  signal(SIGSEGV, handler);
+  signal(SIGTERM, handler);
+  signal(SIGINT, handler);
+  signal(SIGILL, handler);
+  signal(SIGABRT, handler);
+  signal(SIGFPE, handler);
```

Comment:
Add comments about why these signals need to be handled here? It seems complicated for users who just want to write a hello world program with Ray.